health/micro-ui/web/packages/modules/campaign-manager/src/components/SelectingBoundariesDuplicate.js

```
@@ -233,10 +234,10 @@ const SelectingBoundariesDuplicate = ({ onSelect, formData, ...props }) => {
         <PopUp
           className={"boundaries-pop-module"}
           type={"default"}
-          heading={t("ES_CAMPAIGN_UPDATE_BOUNDARY_MODAL_HEADER")}
+          heading={t(LOCALIZATION.ES_CAMPAIGN_UPDATE_BOUNDARY_MODAL_HEADER)}
           children={[
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)